go.mod

```diff
@@ -91,3 +91,5 @@ replace github.com/openstack-k8s-operators/glance-operator/api => ./api
 // mschuppert: map to latest commit from release-4.13 tag
 // must consistent within modules and service operators
 replace github.com/openshift/api => github.com/openshift/api v0.0.0-20230414143018-3367bc7e6ac7 //allow-merging
+
+replace github.com/openstack-k8s-operators/mariadb-operator/api => github.com/zzzeek/mariadb-operator/api v0.3.1-0.20240213153025-83e6db46eeb8 //allow-merging
```

Comment:
ok I suspect this patch is basically ready to go, we can rebase/update and do another pass once we get the mariadb part merged.